etc/repos.yaml

```
@@ -448,3 +448,4 @@ atmospec: https://github.com/lsst-dm/atmospec.git
 drp_pipe: https://github.com/lsst/drp_pipe.git
 resources: https://github.com/lsst/resources.git
 cell_coadds: https://github.com/lsst-dm/cell_coadds.git
+analysis_drp: https://github.com:lsst/analysis_drp.git
```

```suggestion
analysis_drp: https://github.com/lsst/analysis_drp.git
```
I think you will need to temporarily have your ticket branch specified here in order for the Jenkins build to succeed.